.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+<!--
+Thank you for contributing to Kong/charts. Please read through our contribution
+guidelines to understand our review process: https://github.com/Kong/charts/blob/master/CONTRIBUTING.md
+
+When updates to your PR are requested, please add new commits and do not squash the
+history. This will make it easier to identify new changes. The PR will be squashed
+when it is merged.
+-->
+
+#### What this PR does / why we need it:
+
+#### Which issue this PR fixes
+*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
+  - fixes #
+
+#### Special notes for your reviewer:
+
+#### Checklist
+[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
+- [ ] PR is based off the current tip of the `next` branch
+- [ ] New or modified sections of values.yaml are documented in the README.md
+- [ ] Title of the PR and commit headers start with chart name (e.g. `[kong]`)
```

Comment:
Please add a checklist item to ensure that the PR is targeting the next branch and not master.